### ISSUE TITLE
Grouping Variable Options

### ISSUE THREAD

**Issue by L-Haynes** [ABCD-STUDY/DEAP#87]
Currently the grouping variable dropdown menu in Analyze only allows you to select one of five predefined options (high.educ, household.income etc.).  It would be nice to be able to enter any categorical variable here, including custom variables created using Extend.

**Comment by HaukeBartsch** [comment]
Yes, that would be a good idea. Hope we can make this happen soon.

**Comment by Mosqidiot** [comment]
I enabled all the variables that are factor (categorical) type as group options. Please test around on abcd-deap.ucsd.edu.

**Comment by HaukeBartsch** [comment]
Fangzhou, please leave the "No Grouping" section at the beginning of the list. Also, are you able to add the beginning of the item description (up to N=25 characters)? That would make it a little bit easier to identify the correct item without having to search again.
As an enhancement it would be great if we can group these items by domain. That is to have a domain String per section of measures (like the Extend grouping feature for public). A simple way to get some grouping information is to use the NDA column for each instrument.
